tests/utils/IntPointTest.cpp

```diff
@@ -0,0 +1,53 @@
+//Copyright (c) 2015 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#include "IntPointTest.h"
+
+#include <../src/utils/intpoint.h>
+
+namespace cura
+{
+    CPPUNIT_TEST_SUITE_REGISTRATION(IntPointTest);
+
+void IntPointTest::setUp()
+{
+    //Do nothing.
+}
+
+void IntPointTest::tearDown()
+{
+    //Do nothing.
+}
+
+void IntPointTest::test()
```

Comment:
Don't call your test "test". Say what it tests.

Reply:
renamed to testRotationMatrix